.changeset/healthy-rice-shout.md

```diff
@@ -0,0 +1,5 @@
+---
+'@xstate/react': patch
+---
+
+Exposing a separate entrypoint for @xstate/react/fsm
```

Comment:
```suggestion
Added an explicit entrypoint for `@xstate/react/fsm` which you can use instead of `@xstate/react/lib/fsm`. This is the only specifier that will be supported in the future - the other one will be dropped in the next major version.

```diff
-import { useMachine } from '@xstate/react/lib/fsm'
+import { useMachine } from '@xstate/react/fsm'
```
```